CHANGELOG.md

```diff
@@ -75,6 +75,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Bump `opencensus-contrib-http-util` from 0.18.0 to 0.31.1 ([#3633](https://github.com/opensearch-project/OpenSearch/pull/3633))
 - Bump `geoip2` from 3.0.1 to 3.0.2 ([#5103](https://github.com/opensearch-project/OpenSearch/pull/5103))
 ### Changed
+- Revert changes in AbstractPointGeometryFieldMapper ([#5246](https://github.com/opensearch-project/OpenSearch/pull/5246))
```

Comment:
I think we can omit a changelog entry since it doesn't change any user-facing functionality. I'll add the `skip-changelog` label to the PR.